src/main/java/com/eucalyptus/tests/awssdk/S3CorsTests.java

```diff
@@ -1,16 +1,20 @@
 package com.eucalyptus.tests.awssdk;
 
+//LPT To switch between testing against Eucalyptus and AWS,
+//LPT (un)comment the code identified by LPTEuca and LPTAWS.
```

Comment:
@nephomaniac @shaon Is there a better way in n4j to handle switching between AWS and Euca testing than changing the code in this test? See LPT refs below.